server/resolvers/TestResultOperations/saveTestResultCommon.js

```diff
@@ -95,23 +94,31 @@ const assertTestResultIsValid = newTestResult => {
         }
     };
 
-    const checkUnexpectedBehavior = unexpectedBehavior => {
-        if (
-            (!!unexpectedBehavior.otherUnexpectedBehaviorText &&
-                unexpectedBehavior.id !== 'OTHER') ||
-            (!unexpectedBehavior.otherUnexpectedBehaviorText &&
-                unexpectedBehavior.id === 'OTHER')
-        ) {
+    const checkUnexpectedBehavior = (
+        unexpectedBehavior,
+        unexpectedBehaviorNote
+    ) => {
+        if (!unexpectedBehaviorNote && unexpectedBehavior.id === 'OTHER') {
```

Comment:
Non-blocking: Are we still expecting a behavior note to be filled when OTHER? I thought the note would be decoupled from all undesirable behaviors.